claim-check-pattern/ReadME.md

```diff
@@ -0,0 +1,131 @@
+---
```

Comment:
Can you check the filename? It should be consistent `README.md`

Reply:
@iluwatar renamed.